aiocouch/couchdb.py

```diff
@@ -91,13 +91,13 @@ async def create(
 
         """
         db = Database(self, id)
-        if not await db._exists():
+        try:
             await db._put(**kwargs)
-            return db
-        elif exists_ok:
-            return db
-        else:
-            raise PreconditionFailedError(f"The database '{id}' does already exist.")
+        except PreconditionFailedError as e:
+            if not exists_ok:
```

Comment:
This check is not sufficient, as the PreconditionFailedError includes two other errors as well, which we would circumvent by doing it this way.